devtools/maven/src/main/java/io/quarkus/maven/BuildFileMojoBase.java

```diff
@@ -137,6 +131,39 @@ public void execute() throws MojoExecutionException {
         }
     }
 
+    private QuarkusPlatformDescriptor loadPlatformDescriptor(final MavenArtifactResolver mvn, final MessageWriter log,
+            Artifact descrArtifact) {
+        return QuarkusJsonPlatformDescriptorResolver.newInstance()
+                .setArtifactResolver(new BootstrapAppModelResolver(mvn))
+                .setMessageWriter(log)
+                .resolveFromJson(descrArtifact.getFile().toPath());
+    }
+
+    private Artifact resolveJsonOrNull(MavenArtifactResolver mvn, String bomGroupId, String bomArtifactId, String bomVersion) {
+        Artifact jsonArtifact = new DefaultArtifact(bomGroupId, bomArtifactId, null, "json", bomVersion);
+        try {
+            jsonArtifact = mvn.resolve(jsonArtifact).getArtifact();
+        } catch (Exception e) {
+            if (getLog().isDebugEnabled()) {
+                getLog().debug("Failed to resolve JSON descriptor as " + jsonArtifact);
+            }
+            jsonArtifact = new DefaultArtifact(bomGroupId, bomArtifactId + "-descriptor-json", null, "json",
+                    bomVersion);
+            try {
+                jsonArtifact = mvn.resolve(jsonArtifact).getArtifact();
+            } catch (Exception e1) {
+                if (getLog().isDebugEnabled()) {
+                    getLog().debug("Failed to resolve JSON descriptor as " + jsonArtifact);
```

Comment:
Add the exception stacktrace to the debug message

Reply:
TBH, I'd rather remove this logging altogether. Given that we are trying every imported BOM here, if it happens to be not a platform BOM is pretty much a usual situation. IMO, logging a stacktrace is going to make look scarier than it actually is.
The reason I'd consider removing this log message is that if you enable debug, the resolver itself is going to be pretty verbose about what's been tried and what didn't work.